bats/tests/preferences/verify-paths.bats

```diff
@@ -0,0 +1,98 @@
+# Test case 30
+
+setup() {
+    load '../helpers/load'
```

Comment:
Since my RD_LOCATION changes have been merged, the `load helpers/load` call must be made at the top-level and not inside `setup()`.

```suggestion
load '../helpers/load'

setup() {
```